data/tribes/buildings/productionsites/frisians/charcoal_burners_house/init.lua

```diff
@@ -78,18 +78,18 @@ tribes:new_productionsite_type {
          -- TRANSLATORS: Completed/Skipped/Did not start making a charcoal stack because ...
          descname = _"making a charcoal stack",
          actions = {
-            "return=skipped unless economy needs coal",
+            "return=skipped when economy needs log",
             "return=failed unless site has log:3",
-            "callworker=make_stack",
```

Comment:
Is make_stack still being called somewhere?

Reply:
yes 2 lines below. Find_pond is a helper function.  
It was suggested to first consume and then make a stack in an empty pond. To ensure we don't consume if there isn't the possibility on the map to really produce I added this helper just to check whether there is a free pond to burn coal in. by this I could move make stack after consuming the wood.